.travis.yml

```diff
@@ -11,8 +11,11 @@ python:
 cache:
   pip: true
 
+# TODO: if deployment to PyPi is desired, change 'DEPLOY_PYPI' to "true",
```

Comment:
Once the hash is pasted into .travis.yml, please delete this entire TODO comment block.